### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by jrmerwin**
Three Android documents have been added: a general android prerequisites doc, a tutorial for the demo app DL4JIrisClassifierDemo, and a tutorial for the demo app DL4JImageRecognitionDemo.

### CODE REVIEW

File: android-DL4JImageRecognitionDemo.md [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+## <a name="head_link1">Setting the Dependencies</a>
+Deeplearning4J applications requires application specific dependencies in the build.gradle file. The Deeplearning library in turn depends on the libraries of ND4J and OpenBLAS, thus these must also be added to the dependencies declaration. Starting with Android Studio 3.0, annotationProcessors need to be defined as well, thus dependencies for either -x86 or -arm processors should be included, depending on your device, if you are working in Android Studio 3.0 or later. Note that both can be include without conflict as is done in the example app.
+```java
+	compile 'com.android.support:appcompat-v7:27.0.2'
+        compile 'com.android.support:design:27.0.2'
+        compile 'org.deeplearning4j:deeplearning4j-nn:0.9.1'
+        compile 'org.nd4j:nd4j-native:0.9.1'
+        compile 'org.nd4j:nd4j-native:0.9.1:android-x86'
+        compile 'org.nd4j:nd4j-native:0.9.1:android-arm'
+        compile 'org.bytedeco.javacpp-presets:systems-platform:1.4'
```
Comment (saudet) [inline android-DL4JImageRecognitionDemo.md]: We actually should just need `compile 'org.bytedeco:javacpp:1.4'` here.

File: android-DL4JImageRecognitionDemo.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+	exclude 'META-INF/DEPENDENCIES.txt'
+	exclude 'META-INF/LICENSE'
+	exclude 'META-INF/LICENSE.txt'
+	exclude 'META-INF/license.txt'
+	exclude 'META-INF/NOTICE'
+	exclude 'META-INF/NOTICE.txt'
+	exclude 'META-INF/notice.txt'
+	exclude 'META-INF/INDEX.LIST'
+ 
+	exclude 'org/bytedeco/javacpp/windows-x86/msvcp120.dll'
```
Comment (saudet) [inline android-DL4JImageRecognitionDemo.md]: Is this still happening with 0.9.1? It shouldn't be...

File: android-DL4JImageRecognitionDemo.md (hunk 1 above)
Comment (merwijas) [reply android-DL4JImageRecognitionDemo.md]: Yes you're right. I changed that dependency and it worked fine.

File: android-DL4JImageRecognitionDemo.md (hunk 2 above)
Comment (jrmerwin) [reply android-DL4JImageRecognitionDemo.md]: Right again, those exclusions are no long needed. I will remove them from the code.

Comment (saudet) [reply android-DL4JImageRecognitionDemo.md]: Actually, they might pop up again if we add a dependency deeplearning4j-core. What would be better is to exclude all the "-platform" artifacts like ` exclude group: 'org.bytedeco.javacpp-presets', module: 'opencv-platform' ` and add manually the ones we need like `org.bytedeco.javacpp-presets:opencv:version:android-arm`. That's the kind of thing we'd ideally want a gradle-javacpp plugin do for us...

**Comment by jrmerwin** [comment]
I removed the exclusions that are not needed in either of the demos. Both tutorials have instructions on how to add exclusions if duplicate file errors occur with a different dependency.

File: android-DL4JImageRecognitionDemo.md (hunk 2 above)
Comment (jrmerwin) [reply android-DL4JImageRecognitionDemo.md]: I removed the unnecessary exclusions. There are instructions in that section of all three tutorials that explain how to add  exclusions if any errors arise from duplicate files based on the dependencies used. Does that sound OK?

File: android-DL4JIrisClassifierDemo.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+## <a name="head_link1">Setting the Dependencies</a>
+Deeplearning4J applications require several dependencies in the build.gradle file. The Deeplearning library in turn depends on the libraries of ND4J and OpenBLAS, thus these must also be added to the dependencies declaration. Starting with Android Studio 3.0, annotationProcessors need to be defined as well, requiring dependencies for -x86 or -arm processors. 
+```java
+	compile 'com.android.support:appcompat-v7:27.0.2'
+        compile 'com.android.support:design:27.0.2'
+        compile 'org.deeplearning4j:deeplearning4j-nn:0.9.1'
+        compile 'org.nd4j:nd4j-native:0.9.1'
+        compile 'org.nd4j:nd4j-native:0.9.1:android-x86'
+        compile 'org.nd4j:nd4j-native:0.9.1:android-arm'
+        compile 'org.bytedeco.javacpp-presets:systems-platform:1.4'
```
Comment (saudet) [inline android-DL4JIrisClassifierDemo.md]: Please replace all the `'org.bytedeco.javacpp-presets:systems-platform:1.4'` with `'org.bytedeco:javacpp:1.4'`

**Comment by saudet** [comment]
Yes, that's fine, thanks! But we should *really* document how to exclude the "-platform" artifacts and reinclude the needed artifacts for Android. You're going to run into this when trying to get this demo with images actually running...

**Comment by crockpotveggies** [comment]
If there are further updates please open a new pull request. Existing one is merged. Thanks @jrmerwin !

**Comment by Yu-Hang** [comment]
@jrmerwin Where do I look for these docs?

**Comment by saudet** [comment]
@Yu-Hang Here at https://deeplearning4j.org/android but we still need to fix the formatting and update the examples @jrmerwin

**Pull request merged**